ops/services/alerts/app_service_metrics/main.tf

```diff
@@ -144,7 +144,7 @@ ${local.skip_on_weekends}
 
   trigger {
     operator  = "GreaterThan"
-    threshold = 9
+    threshold = 24
```

Comment:
I'm about to submit a PR to exclude some of the exceptions we talked about earlier, which is probably going to reduce the exceptions seen by this alert a bit - does that change your threshold at all?

Reply:
Sounds good - we can put this on hold for a bit and see how that one plays out!